README.future.md

```diff
@@ -2,99 +2,563 @@
 
 **This README is a draft related to an upcoming release of the node pinecone client. Everything in this document is subject to change.**
 
-## Client Initialization
+# Pinecone Node.js Client
 
-To get started, you need to create a client that specifies your Pinecone credentials. These values are found in Pinecone's web-based developer console.
+This is the official Node.js client for Pinecone, written in TypeScript.
```

Comment:
Is it useful to add a section that mentions what node versions are supported? Or the minimum node version needed to run?